presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpch.yaml

```diff
@@ -0,0 +1,37 @@
+datasource: presto
+query-names: presto/tpch/${query}.sql
+runs: 6
```

Comment:
https://github.com/prestodb/benchto/issues/37 should solve the issue. For now I think benchmarks could serve as a template